### ISSUE TITLE
Exclude Directory?

### ISSUE THREAD

**Issue by misilot** [erclu/check-crlf#10]
Is there a way to exclude a directory? For example, I've got a php application that the libraries are included under "vendors/" in the codebase and those aren't something I would want to actually check.

Thanks!

**Comment by jmckenna** [comment]
Yes, here is a working example: https://github.com/MapServer/MapServer/blob/main/.github/workflows/check-crlf.yml
